frontend/src/components/OdhAppCard.tsx

```diff
@@ -142,37 +161,34 @@ const OdhAppCard: React.FC<OdhAppCardProps> = ({ odhApp }) => {
 
   const popoverBodyContent = (hide: () => void) => (
     <div>
-      {isInternalRouteIntegrationsApp(odhApp.spec.internalRoute) ? (
-        <>
-          {odhApp.spec.error ? `${capitalize(odhApp.spec.error)}.` : ''} Contact your administrator.
-        </>
-      ) : (
-        <>
-          Subscription is no longer valid. To validate click&nbsp;
-          <Button
-            isInline
-            variant="link"
-            onClick={() => {
-              hide();
-              setEnableOpen(true);
-            }}
-          >
-            here.
-          </Button>
-          To remove card click&nbsp;
-          <Button
-            isInline
-            variant="link"
-            onClick={() => {
-              hide();
-              removeApplication();
-            }}
-          >
-            here
-          </Button>
-          .
-        </>
+      {isInternalRouteIntegrationsApp(odhApp.spec.internalRoute) && odhApp.spec.error && (
+        <>{`${capitalize(odhApp.spec.error)}.`}</>
       )}
+      <p>
+        Subscription is no longer valid. To validate click&nbsp;
+        <Button
+          isInline
+          variant="link"
+          onClick={() => {
+            hide();
+            setEnableOpen(true);
+          }}
+        >
+          here.
+        </Button>
+        To remove card click&nbsp;
+        <Button
```

Comment:
Small UX nitpick with the grammer. There is no space after the `here.` 

```suggestion
        >
          here
        </Button>
        . To remove card click&nbsp;
        <Button
```

Reply:
Updated

Reply:
So you made the "." part of the hyperlink but there is no space before it still 
![image](https://github.com/user-attachments/assets/2c60404b-ee33-4bff-9c33-a70eeccfd7c1)

adding a `&nbsp;` should fix it (or use the original commit suggestion too)

```suggestion
        >
          here.
        </Button>
        &nbsp;To remove card click&nbsp;
        <Button
```

Reply:
Sorry, my bad. Updated.